src/databricks/labs/remorph/reconcile/constants.py

```diff
@@ -60,11 +63,9 @@ class ReportType(AutoName):
 class ThresholdMode(AutoName):
     PERCENTILE = "percentile"
```

Comment:
nit: Shouldn't this is percentage not percentile?

Reply:
Done.changes it to percentage